packages/fxa-content-server/app/scripts/views/settings/subscription.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+import Cocktail from 'cocktail';
+import FormView from '../form';
+import SettingsPanelMixin from '../mixins/settings-panel-mixin';
+import Template from 'templates/settings/subscription.mustache';
+
+const View = FormView.extend({
+  template: Template,
+  className: 'subscription',
+  viewName: 'settings.subscription',
+
+  events: {
+    'click button': 'submit',
+  },
+
+  initialize (options = {}) {
+    this._config = Object.assign({}, {
```

Comment:
See the [comment above about removing the hard coded config](https://github.com/mozilla/fxa/pull/940/files#diff-0370004e39359bfe3201953deacddcacR432) here and instead relying upon the [config passed in from configuration.js](https://github.com/mozilla/fxa/pull/940/files#diff-e6c30fd24af1b7b4b7d351ad516d6092R597).